tests/e2e/e2e_workflow.spec.js

```diff
@@ -155,10 +155,11 @@ describe('Basic end-to-end Workflow', function () {
     expect(existsSync(path.join(process.cwd(), 'dist/app-concat.js'))).to.be.equal(false);
   });
 
-  it('lints', () => {
+  it('lints', function () {
```

Comment:
In some cases tests were failing because of timeout, in that way we can use `this.timeout`.